notebook_utils/berdl_notebook_utils/postgres/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""PostgreSQL integration modules for BERDL."""
```

Comment:
The user should not have any credentials for postgres

Reply:
We discussed in slack and will try the hive client

Reply:
👍 thanks - using hive client to retrieve table info now.